Source/OIDAuthorizationService.m

```diff
@@ -558,9 +585,14 @@ + (void)performRegistrationRequest:(OIDRegistrationRequest *)request
       }
 
       // not an OAuth error, just a generic server error
+      NSString *errorDescription =
+          [NSString stringWithFormat:@"Non-200/201 HTTP response (%d) making registration request "
+                                     "to '%@'.",
+                                     (int)HTTPURLResponse.statusCode,
```

Comment:
The only reason I'm mentioning this is because I seem to recall there being an optional warning about loss of precision here on macOS, iOS, tvOS since [NSInteger is a long on those platforms](https://developer.apple.com/documentation/objectivec/nsinteger?language=objc).

Reply:
There was a warning about loss of precision, which is why I added the cast here. I considered casting to (long) instead, but HTTP status codes well fit within `int`, so it seemed fine to go that way.